ui/app/serializers/connector.js

```diff
@@ -7,9 +7,9 @@ const CONNECTOR_TYPE_MAP = {
 
 export default class ConnectorSerializer extends ApplicationSerializer {
   serialize(snapshot) {
-    const configSettings = this._replaceKeys(
+    const configSettings = super._replaceKeys(
       snapshot.record.config.settings,
```

Comment:
I noticed that `config.settings` is guard checked against in the `normalize` method via `hash.config?.settings` -- would it make sense to do the same here as well?

Reply:
On serialization of a connector, the config object will always be present due to the connector model and how we create/update connectors so its not totally necessary, but I don't think it could hurt

Reply:
I see, thank you for the outline and I agree with you that we could then just go either way here -- I think it's best to be your call!